packages/ui-react/src/containers/AccountModal/forms/Login.tsx

```diff
@@ -29,13 +29,21 @@ const Login: React.FC<Props> = ({ messageKey }: Props) => {
 
   const queryClient = useQueryClient();
 
+  // temporary state to show success message
+  const [successState, setSuccessState] = React.useState(false);
+
   const loginSubmitHandler: UseFormOnSubmitHandler<LoginFormData> = async (formData, { setErrors, setSubmitting, setValue }) => {
     try {
+      setSubmitting(true);
```

Comment:
No need to call this. The `useForm` submit handler already does this under the hood.

```suggestion
```

Reply:
Good to know, i'll remove it in the next commit!
- [ ] Remove redundant setSubmitting.